docs/api.md

```diff
@@ -1016,6 +1019,10 @@ event instead.
 
 Emitted when the server changes any of the game properties.
 
+#### "resourcePack" (url, hash)
```

Comment:
No, it emits an object that contains the url and hash, it only has one parameter

Reply:
I liked the idea of an object here like before, its more future proof and can be expanded later on to add fields like pack size. What you can do is update the doc to show that it’s an object instead of parameters.